README.md

```diff
@@ -279,6 +292,22 @@ Kafka Cluster Connection Details (`kafka-lag-exporter.clusters[]`)
 | `admin-client-properties` | `{}`        | No       | A map of key value pairs used to configure the `AdminClient`. See the [Admin Config](https://kafka.apache.org/documentation/#adminclientconfigs) section of the Kafka documentation for options.          |
 | `labels`                  | `{}`        | No       | A map of key value pairs will be set as additional custom labels per cluster for all the metrics in prometheus.                                                                                           |
 
+Redis Details (`kafka-lag-exporter.redis{}`)
+
+| Key          | Default                | Required | Description                                                                                                         |
+|--------------|------------------------|----------|---------------------------------------------------------------------------------------------------------------------|
+| `enabled`    | `"false"`              | No       | Switch to enable or disable the Redis table (if enabled, the memory tables will not be created).                    |
+| `database`   | `0`                    | No       | Redis database number.                                                                                              |
+| `host`       | `"localhost"`          | No       | Redis server to use.                                                                                                |
+| `port`       | `6379`                 | No       | Redis port to use.                                                                                                  |
+| `timeout`    | `60`                   | No       | Redis connection timeout.                                                                                           |
+| `prefix`     | `"kafka-lag-exporter"` | No       | Prefix used by all the keys.                                                                                        |
+| `separator`  | `":"`                  | No       | Separator used to build the keys.                                                                                   |
+| `resolution` | `"1 minute"`           | No       | Resolution of the lookup table. Last point will get updated if the collection interval is less than the resolution. |
```

Comment:
Is there a reason for this not to be the same as the `poll-interval` configuration?

Reply:
@guillaume-roland-cd Can you comment on this ^

Reply:
That's actually a good question, and I even wonder if having that option is useful or not.
Right now, I don't see the value of having a resolution higher than the poll-interval, as it mean we will collect metrics knowing we will discard them later.
Maybe I should just remove the resolution setting?

Reply:
SGTM

Reply:
Alright, I removed everything related to that feature.